neptune/new/project.py

```diff
@@ -30,9 +35,21 @@ class Project:
        https://docs.neptune.ai/api-reference/project
     """
 
-    def __init__(self, _id: str, backend: HostedNeptuneBackend):
-        self._id = _id
-        self._backend = backend
+    def __init__(
+        self,
+        _id: str,
+        backend: NeptuneBackend,
+        op_processor: OperationProcessor,
+        background_job: BackgroundJob,
+        lock: threading.RLock,
+        workspace: str,
+        project_name: str,
+    ):
+        super().__init__(
+            _id, backend, op_processor, background_job, lock, project_id=_id
+        )
+        self._workspace = workspace
```

Comment:
Maybe `project_name` and `workspace` should be shared via `AttributesContainer` with Runs? Only type verifycation for non-null seems to be reasonable.

Reply:
done, I think